github-upload-public-key/run.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -e
+
+CODER_ACCESS_URL="${CODER_ACCESS_URL}"
+CODER_OWNER_SESSION_TOKEN="${CODER_OWNER_SESSION_TOKEN}"
+CODER_EXTERNAL_AUTH_ID="${CODER_EXTERNAL_AUTH_ID}"
+GITHUB_API_URL="${GITHUB_API_URL}"
+
+if [ -z "$CODER_ACCESS_URL" ]; then
+  echo "No coder access url specified!"
```

Comment:
For all of these I might put the env var it's looking for instead. Might reduce some confusion.